manifests/crds/cpet.belastingdienst.nl_paas.yaml

```diff
@@ -42,12 +42,17 @@ spec:
               capabilities:
                 additionalProperties:
                   properties:
+                    custom_fields:
+                      additionalProperties:
+                        type: string
+                      description: Custom fields to configure this specific Capability
+                      type: object
                     enabled:
                       description: Do we want an ArgoCD namespace, default false
```

Comment:
This is not part of the changes but caught my eyes. Can you add a suitable description here?

Reply:
Yeah, I noticed some more leftovers from implementing configurable capabilities.
I have added a separate new PR for it:
https://github.com/belastingdienst/opr-paas/pull/179